### PR TITLE
Fix: internationalization of french part

### PR DESCRIPTION
**Pull request by SirineMhedhbi**
Related to https://github.com/agroportal/project-management/issues/309

### Changes 

* internationalization of ontologies helpers

*  Internationalization of remaining files

* update of en.yml file (remove the duplacted keys)

* update of fr.yml file

* fix the bug of missing translate

### CODE REVIEW

**Comment by syphax-bouazzouni** [comment]
Delpoyed to Stageportal, to verify and complete @SirineMhedhbi , @jonquet

File: config/locales/fr.yml [hunk 1]
```diff
@@ -158,138 +459,254 @@ fr:
...
       cite_us: Citez-nous
     footer:
       products: Produits
       ontoportal: OntoPortal
       release_notes: Notes de version
       api: API
       sparql: SPARQL
-      support: Assistance
-      contact_us: Contactez nous
+      support: support
```
Comment (Bilelkihal) [inline config/locales/fr.yml:478]: 'S' in uppercase

File: config/locales/fr.yml [hunk 2]
```diff
@@ -158,138 +459,254 @@ fr:
...
   login:
-    enter_email: Saisissez votre nom d'utilisateur
-    enter_password: Saisissez votre mot de passe
+    invalid_account_combination: "Combinaison nom de compte/mot de passe invalide"
+    authentication_failed: "L'authentification %{provider} a échoué"
+    admin_logged_out: Déconnecté <b>%{old_user}</b>, retourné à <b>%{user}</b>
+    user_logged_out: Vous vous êtes déconnecté avec succès
+    try_again_notice: ". Veuillez réessayer."
+    reset_password_again: ". Veuillez réinitialiser votre mot de passe à nouveau."
+    custom_ontology_set: "L'affichage est maintenant basé sur votre <a href='/account#custom_ontology_set'>Ensemble d'Ontologies Personnalisé</a>."
```
Comment (Bilelkihal) [inline config/locales/fr.yml:504]: We need to keep html elements in the .haml file to keep code modification easier

File: config/locales/fr.yml [hunk 3]
```diff
@@ -344,73 +801,701 @@ fr:
...
+  select_ontologies_list: Sélectionner des ontologies
   show_advanced_options: Afficher les options avancées
   specialization: Spécialisation
   temporality: Temporalité
   text: Texte
   type: Type
-  umls_sem_type: Type Semantique UMLS
-  view_fair_scores_definitions: Voir les définition des scores de FAIRness
+  umls_sem_type: Type Sémantique UMLS
+  view_fair_scores_definitions: "Vous voyez le score d'évaluation de l'équité FAIR pour cette ontologie dans AgroPortal. Les scores FAIR sont calculés avec la méthodologie O'FAIRe. Plus de détails ici : https://github.com/agroportal/fairness"
```
Comment (Bilelkihal) [inline config/locales/fr.yml:947]: Keep this email in the .haml file

File: config/locales/fr.yml [hunk 4]
```diff
@@ -344,73 +801,701 @@ fr:
...
+      new_value: Nouvelle valeur
+      old_value: Ancienne valeur
+      class_id: ID de la classe
+      label: Étiquette
+      synonym: Synonyme
+      definition: Définition
+      parent: Parent
+      Save: sauvegarder
+    note_line:
+      alert_text: "Êtes-vous sûr de vouloir supprimer la note <span style='color:red;font-weight:bold;'> ''%{subject}''</span> créée par <span style='color:red;font-weight:bold;'>%{creator}</span>?<br/><b>Cette action NE PEUT PAS être annulée !!!</b>"
```
Comment (Bilelkihal) [inline config/locales/fr.yml:982]: separate the style from the translation text

**Review by Bilelkihal — CHANGES_REQUESTED** [review]

**Pull request merged**